src/styles/components/_component-grid.scss

```diff
@@ -30,8 +30,11 @@ $breakpoint-tablet: 820px;
     width: 100%;
     margin-bottom: 1.25rem;
     background-color: var(--light-background-color);
+    margin: 0;
+    background-color: var(--light-background-color);
```

Comment:
These are repeated properties (we probably missed them during that rebase!), so I would delete them. We want the margin-bottom at line 31.

Reply:
Deleted these!